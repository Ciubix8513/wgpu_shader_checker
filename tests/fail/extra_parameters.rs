use wgpu_shader_checker::include_wgsl;

fn main() {
    include_wgsl!("as;ljghnadk", iawjofi. -0q298o, aola8 * ijkomlacxdkjsdlk)
}
