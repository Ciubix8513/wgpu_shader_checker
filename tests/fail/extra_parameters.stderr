error: unexpected token
 --> tests/fail/extra_parameters.rs:4:32
  |
4 |     include_wgsl!("as;ljghnadk", iawjofi. -0q298o, aola8 * ijkomlacxdkjsdlk)
  |                                ^
